server.js

```diff
@@ -119,10 +119,10 @@ app.get("/api/week", async (req, res) => {
 
 // Serve any static files
 app.use(express.static(path.join(__dirname, "client/build")));
-app.use(
-  "/storybook",
-  express.static(path.join(__dirname, "client/storybook-static"))
-);
+// app.use(
+//   "/storybook",
+//   express.static(path.join(__dirname, "client/storybook-static"))
+// );
```

Comment:
There might be a problem with the run() part of your function and that might be why its not deploying. Did you check your Database names in your .env? right now yours looks like:

> `async function run() {
>   console.log("Connecting to database...");
>   await connect(process.env.DB_URI, process.env.DB_DBNAME);
>   console.log("Connected to database ");`

  while mine looks like this: 

> `async function run() {
  console.log("Connecting to database...");
  await connect(process.env.MONGO_DB_URI, process.env.MONGO_DB_NAME);
  console.log("Connected to database ");`

Reply:
thanks, problem was elserwhere ;)